smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/extension/HealthCenter.java

```diff
@@ -41,6 +41,10 @@ public void report(String source, Throwable cause) {
         failures.add(new ReportedFailure(source, cause));
     }
 
+    public void reportApplicationFailure(String method, Throwable cause) {
+        failures.add(new ReportedFailure("application-" + method, cause));
```

Comment:
Does "application" as a prefix make sense given it's combined with the method name? Is that going to be confusing?

Reply:
Yes and no, I tried with the method name only and found it even more confusing. The `application` prefix indicates it's a code issue.

Reply:
Anyway, let's see and we can revisit it later. The important part is that it's reported.